src/main/java/com/atlauncher/viewmodel/impl/InstancesTabViewModel.java

```diff
@@ -17,74 +17,87 @@
  */
 package com.atlauncher.viewmodel.impl;
 
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
 import com.atlauncher.App;
 import com.atlauncher.data.Instance;
 import com.atlauncher.managers.InstanceManager;
 import com.atlauncher.utils.sort.InstanceSortingStrategy;
 import com.atlauncher.viewmodel.base.IInstancesTabViewModel;
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
 
-import java.util.List;
-import java.util.function.Consumer;
-import java.util.function.Predicate;
-import java.util.regex.Pattern;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
+import io.reactivex.rxjava3.core.Observable;
+import io.reactivex.rxjava3.core.ObservableSource;
+import io.reactivex.rxjava3.schedulers.Schedulers;
+import io.reactivex.rxjava3.subjects.BehaviorSubject;
 
 /**
  * 20 / 11 / 2022
  */
-public class InstancesTabViewModel implements IInstancesTabViewModel, InstanceManager.Listener {
```

Comment:
Can this `InstanceManager.Listener` be removed now, or are there more things it does that it needs to stay around?

Reply:
Off the top of my head, no reason for it to exist anymore. I'll remove it.